### DISCUSSION TITLE
student plan

### DISCUSSION THREAD

**Discussion by Mazen-Osama** [community/community#109698] [GitHub Education]
### Select Topic Area

Question

### Body

I have submitted a request with my id photo for the student plan and i didn't recive any mails or confirmation or anything so what should i do ?

**Comment by luyandadhlamini2** [comment]
#### Please wait for Github to review your submission.

Getting a response on an application may take **4 to 6** days.

**Comment by fraineralex** [comment]
It's not uncommon for the validation process to take some time. In my case, it took over a week. I recommend waiting a bit longer.
